include/yokozuna.hrl

```diff
@@ -263,6 +263,12 @@
 -define(YZ_ENTROPY_LOCK_TIMEOUT,
     app_helper:get_env(?YZ_APP_NAME, anti_entropy_lock_timeout, 10000)).
 
+-define(YZ_ENTROPY_THROTTLE_KEY, yokozuna_aae).
```

Comment:
With the addition of AppName to the riak_core_throttle APIs, we can change this to `aae_enabled` since it'll be prefixed with `yokozuna` as an application name rather than having to be unique.